docs/xxjitserveraotmx.md

```diff
@@ -0,0 +1,49 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:JITServerAOTmx
+
+**(Linux&reg; only)**
+
+This option specifies the maximum memory allocated to the JITServer AOT cache for storing the compiled code and for the associated data structures that are used in the cache's implementation.
+
+## Syntax
+
+        -XX:JITServerAOTmx=<size>
+
+| Setting                 | Effect | Default                                                                            |
+|-------------------------|--------|:----------------------------------------------------------------------------------:|
+| `-XX:JITServerAOTmx=<size>`    | Limits the amount of memory used by the JITServer AOT cache |  300 MB                       |
+
+## Explanation
+
+When the JITServer receives an AOT compilation request, it checks its AOT cache for a compatible compiled method body. If one is not found, the server performs the AOT compilation, sends the response to the client JVM, then stores the compiled method in its local AOT cache, for future use. If the JITServer physical memory is low, multiple requests and storing of the compiled methods can consume a lot of memory and degrade the system's overall performance.
```

Comment:
I would eliminate "If the JITServer physical memory is low" here. Consuming a lot of memory can degrade system's performance regardless of how low the memory at the server is.
Moreover, when the physical memory is low at the server, consuming memory for the AOT cache can lead to poor performance of the client JVMs due to the server inability to perform compilations, or even native OOM crashing the server.